load-test-framework/src/main/resources/command.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+import "google/protobuf/empty.proto";
+import "google/protobuf/timestamp.proto";
+package pubsub;
+
+option java_package = "com.google.pubsub.flic.common";
+option java_outer_classname = "Command";
+
+service LoadtestFramework {
+  rpc StartClient(CommandRequest) returns (google.protobuf.Empty) {
+  }
+}
+
+message CommandRequest {
```

Comment:
Yeah I like that better after thinking about it. More like real-usage for the publisher and subscriber to be different instances.